web/src/components/Callout.tsx

```diff
@@ -0,0 +1,77 @@
+import { ReactElement } from "react";
+
+export type CalloutTypes = "informational" | "conditional" | "warning" | "note";
+
+interface Props {
+  calloutType: CalloutTypes;
+  header?: string;
+  children: string;
+  icon?: "true" | "false";
+}
+
+interface CalloutMappingObj {
+  containerStyling: string;
+  headingStyling: string;
+  headingDefaultText: string;
+  bodyStyling: string;
+  iconStyling: string;
+}
+
+export const calloutComponentMapping: Record<string, CalloutMappingObj> = {
+  informational: {
+    containerStyling: "bg-accent-cool-lightest",
+    headingStyling: "text-accent-cool-more-dark",
+    headingDefaultText: "Keep in mind:",
```

Comment:
[sand] If content intends to use the default texts, for all variants, widely as a default we may want it to be available in the CMS.